### DISCUSSION TITLE
[Feature]:  Predefined Properties with type Object and Content Type object should appear in document and PrintPage

### DISCUSSION THREAD

**Discussion by syncrasy-git** [pimcore/pimcore#11392] [Q&A / Support]
### Feature description

Hello, I have a query. I have created Predefined Properties with type Object and Content Type object, but it does not reflect/appear in document type PrintPage Properties

**Comment by blankse** [comment]
@rajaryann 
https://pimcore.com/docs/pimcore/current/Development_Documentation/Tools_and_Features/Properties.html

> Content-Type: Defines for which element-type (document, asset or object) the property should be available.

So if you set the Content-Type to object it should not be available for documents. You need to set the Content Type to document.
